viewer/packages/3d-overlays/src/image360Icon.frag

```diff
@@ -0,0 +1,18 @@
+precision highp float;
+
+uniform sampler2D map;
+uniform vec3 colorTint;
+uniform float collectionOpacity;
+
+in vec3 vColor;
+
+out vec4 fragmentColor;
+
+void main() {
+  vec2 coord = gl_PointCoord - vec2(0.5);
+  vec4 textureSample = texture(map, gl_PointCoord); 
+  if (textureSample.a <= 0.1) {
```

Comment:
When set to 0.0 shadow gradient that is created for overlay texture creates a "border" around circle overlay that results in incorrect transparency look from specific views on points. See screenshots.

**With 0.1:**
![Screenshot 2023-05-05 at 16 05 13](https://user-images.githubusercontent.com/45129444/236487124-2e036dd6-43e3-47a3-8d66-4cf9b56c1b2f.png)

**With 0.0:**
![Screenshot 2023-05-05 at 16 04 43](https://user-images.githubusercontent.com/45129444/236487262-8fd4c0bc-3995-43f2-a5bb-5a170449d2d6.png)

Reply:
One way of solving this "glowing" issue would be just by changing the texture of the overlay as we haven't really agreed on some specific design. If the circle have a border instead of the shadow everything looks much cleaner. Here is another proposition for the design, we can decide what is better later, probably in a separate PR.

Slightly different design:
![Screenshot 2023-05-05 at 17 21 26](https://user-images.githubusercontent.com/45129444/236500178-978585e6-b2e6-4b16-8ed2-928e80c44f06.png)
![Screenshot 2023-05-05 at 17 21 42](https://user-images.githubusercontent.com/45129444/236500180-b481427d-022f-4d58-a35e-388321e89f52.png)

Reply:
I see - In that case I think I'd advocate for having a border instead of a shadow effect. I suppose we will discuss this later